src/Relude/String/Conversion.hs

```diff
@@ -239,6 +245,23 @@ instance ToText LText where
     toText = LT.toStrict
     {-# INLINE toText #-}
 
+-- | @since 0.6.0.0
+instance ToMsg ToText ByteString Text => ToText ByteString where
+    toText :: ByteString -> Text
+    toText = error "unreachable"
+
+-- | @since 0.6.0.0
+-- QUESTION: 'LByteString' will be shown either as 'ByteString' or
+-- 'Data.ByteString.Lazy.Internal.ByteString'. Should it be type-level string?
```

Comment:
Why so?

Reply:
Because `LByteString` is a type alias and compiler doesn't know about type aliases and shows the concrete type in type errors... That's why we all see `[Char]` in error messages instead of `String`...

Reply:
Oh, thanks for the clarification!
Probably that makes sense to introduce type level string, otherwise, it could be confusing, what do you think?

Reply:
@vrom911 Yes, I agree that it would be much nicer to see `LByteString` instead of `Data.ByteString.Lazy.Internal.ByteString` :+1: